### PR TITLE
21 login page tests

### PR DESCRIPTION
**Pull request by tstefacek**

### CODE REVIEW

File: front-end/package-lock.json [hunk 1]
```diff
@@ -8470,218 +8470,29 @@
       }
     },
     "jest-watch-typeahead": {
-      "version": "1.1.0",
-      "resolved": "https://registry.npmjs.org/jest-watch-typeahead/-/jest-watch-typeahead-1.1.0.tgz",
-      "integrity": "sha512-Va5nLSJTN7YFtC2jd+7wsoe1pNe5K4ShLux/E5iHEwlB9AxaxmggY7to9KUqKojhaJw3aXqt5WAb4jGPOolpEw==",
+      "version": "0.6.5",
```
Comment (krgirard33) [inline front-end/package-lock.json]: Why is jest-watch-typeahead switching from 1.1.0 to 0.6.5? Also, latest version of jest-watch-typeahead is 2.2.2... so it is even further back from current version.

File: front-end/package.json [hunk 2]
```diff
@@ -38,5 +38,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "jest-watch-typeahead": "^0.6.5"
```
Comment (krgirard33) [inline front-end/package.json:44]: jest-watch-typeahead: Why 0.6.5, and not 1.1.0, or latest version 2.2.2?

**Review by krgirard33 — CHANGES_REQUESTED** [review]
Questions about jest-watch-typeahead version, and not sure how that will impact all the test.

File: front-end/package-lock.json (hunk 1 above)
Comment (tstefacek) [reply front-end/package-lock.json:8752]: Not 100% sure, but I went ahead and installed the latest version and pushed the changes.

Comment (tstefacek) [reply front-end/package-lock.json]: I updated Jest to the most recent version and that line of code stayed the same. All the tests pass, and everything functions as it should. If there's a way and a reason for me to update that particular item, please let me know!

**Review by krgirard33 — APPROVED** [review]

**Review by krgirard33 — APPROVED** [review]
Looks good and makes sense

**Review by krgirard33 — APPROVED** [review]
Looks good & makes sense

**Review by krgirard33 — APPROVED** [review]
Looks good & makes sense

**Review by krgirard33 — APPROVED** [review]
Looks good & makes sense

File: front-end/src/App.js [hunk 3]
```diff
@@ -1,11 +1,10 @@
 import { useState } from 'react';
 import './App.css';
```
Comment (Ballarja) [inline front-end/src/App.js:2]: looks like this change could cause a conflict

import Registration from "./components/layout/SignUp";
import Landing from "./components/layout/Landing";
import Login from "./components/layout/LoginPage";

File: front-end/src/App.test.js [hunk 4]
```diff
@@ -1,8 +1,6 @@
 import { render, screen } from '@testing-library/react';
```
Comment (Ballarja) [inline front-end/src/App.test.js:1]: test look good

**Review by Ballarja — APPROVED** [review]
cleaned up code and everything is looking good

**Pull request merged**